Footer/Footer 11 lift blue.html

```diff
@@ -0,0 +1,95 @@
+<footer class="bg-blue-600 antialiased">
```

Comment:
To handle dark/light theme changes. You can use this.
`bg-white dark:bg-blue-600`

Reply:
Thank you for your suggestion. I can do that too but this component has a black variant too